modules/storages/spec/features/storages/admin/create_storage_spec.rb

```diff
@@ -91,6 +91,9 @@
 
       aggregate_failures "General information" do
         within_test_selector("storage-general-info-form") do
+          expect(page).to have_enterprise_banner(:corporate)
+          expect(page).to have_selector('option[disabled=disabled][value=oauth2_sso]') # expect SSO option to be disabled
```

Comment:
:exclamation: Help wanted, this expectation looks pretty ugly :exclamation: 

What I'd prefer to do is something along the lines of (made up code):

```ruby
expect(page).to have_select("Authentication Method").with_disabled_option("Single-Sign-On through OpenID Connect Identity Provider")
```

I.e. something where I can check that the right option in the right select is disabled, but only using the user visible (accessible) names.

This check here works, but that's all the nice things I can say about it.

Reply:
`option:disabled` also works

Reply:
That's fair. It at least makes the CSS selector nicer. But I read that you would also keep using a CSS selector for this expectation? (vs. knowing of a much nicer accessible matcher that I am not aware of)

Reply:
I think the CSS is fine ...If it were a simple form field, we `expect(page).to have_field("something", disabled: true)`. I don't expect it works with options 🤔